src/Our.Umbraco.DocTypeGridEditor/Web/UI/App_Plugins/DocTypeGridEditor/Views/doctypegrideditor.html

```diff
@@ -15,10 +14,18 @@
             </div>
         </div>
     </div>
-    <umb-overlay
-        ng-if="overlay.show"
-        model="overlay"
-        view="overlay.view"
-        position="right">
-    </umb-overlay>
-</div>
\ No newline at end of file
+    <div ng-if="preview">
```

Comment:
@mathiasrando Can you double check this please?
The preview is appearing twice in this file. I think it was due to a merge conflict.

Reply:
Well spotted, Lee. I've cleaned up the markup.